File: Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IRewardedEventListener.java
package com.chartboost.heliumsdk.unity;

/**
 * Interface for sending real-time rewarded events
 */
public interface IRewardedEventListener {
    void DidLoadRewarded(String placementName, int errorCode, String errorDescription);
    void DidShowRewarded(String placementName, int errorCode, String errorDescription);
    void DidCloseRewarded(String placementName, int errorCode, String errorDescription);
    void DidClickRewarded(String placementName, int errorCode, String errorDescription);
    void DidWinBidRewarded(String placementName, String auctionId, String partnerId, double price);
    void DidReceiveReward(String placementName, int reward);
}

File: com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/IRewardedEventListener.kt
package com.chartboost.heliumsdk.unity

/**
 * Interface for sending real-time rewarded events
 */
interface IRewardedEventListener {
    fun DidLoadRewarded(placementName: String, auctionId: String?, partnerId: String, price: Double, error: String)
    fun DidShowRewarded(placementName: String, error: String)
    fun DidCloseRewarded(placementName: String, error: String)
    fun DidClickRewarded(placementName: String, error: String)
    fun DidRecordImpression(placementName: String, error: String)
    fun DidReceiveReward(placementName: String)
}
